tox.ini

```diff
@@ -12,7 +18,8 @@ python =
 [testenv]
 commands = python run_tests.py
 deps =
-    flake8 >= 5.0.0
+    flake85: flake8 >= 5.0.0, < 6.0.0
+    flake86: flake8 >= 6.0.0, < 7.0.0
```

Comment:
I think I would have written these are `flake8-v5` and `flake8-v6`, but I don't have strong feelings.

Reply:
Me too! It's one of my few `tox` experiences in my career; idk if `-` is somehow a special character in `tox`, that it "prepares" tox to start parsing/checking different "environment combinations" or something - or `-` is like any other string (except `{`, `}`, `,` ofc)